vendor/moko/lw004.yaml

```diff
@@ -50,7 +63,7 @@ sensors:
   - battery
   - button
   - gps
-  - light
+  - led
   - vibration
```

Comment:
please review valid sensors if they are not in the list please leave them commented